tee-worker/cli/src/base_cli/commands/litentry/mod.rs

```diff
@@ -15,3 +15,7 @@
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 
 pub mod id_graph_hash;
+pub mod link_identity;
+
+// TODO: maybe move it to use itp_node_api::api_client
```

Comment:
Are you moving it or leaving here as it is ?

Reply:
This is an old comment, I'm leaving it here as it is 😅